charts/cmc-citizen-frontend/values.preview.template.yaml

```diff
@@ -151,6 +151,8 @@ ccd:
       keyVaults:
       environment:
         IDAM_OAUTH2_AW_CLIENT_SECRET: ${ADMIN_WEB_IDAM_SECRET}
+    idam-pr:
```

Comment:
you don't really need this. it's already in ccd-chart-4 we are using

Reply:
I think it is needed